packages/grafana-data/src/themes/createComponents.ts

```diff
@@ -82,6 +86,10 @@ export function createComponents(colors: ThemeColors, shadows: ThemeShadows): Th
     dropdown: {
       background: input.background,
     },
+    popover: {
+      background: colors.mode === 'dark' ? colors.background.secondary : colors.background.primary,
```

Comment:
the reason for the new popover token is to share this theme layering logic

Reply:
nstead of introducing this as component level token we could also introduce it to the base color tokens as a new “background” named “elevated” (so we would have canvas, primary, secondary, elevated) in light theme elevated would be the same as primary. I guess the modal is elevated but there we would likely use primary still as we have the backdrop to help offset it / give it contrast